docs/settings/security-settings.asciidoc

```diff
@@ -362,52 +359,113 @@ To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify wh
 [source,yaml]
 ----------------------------------------
 xpack.security.audit.appender:
-  kind: file
-  path: /path/to/audit.log
+  kind: rolling-file
+  path: ./audit.log
+  policy:
+    kind: time-interval
+    interval: 24h <1>
+  strategy:
+    kind: numeric
+    max: 10 <2>
   layout:
     kind: json
 ----------------------------------------
+<1> Rotate log files every 24 hours
+<2> Keep maximum of 10 log files before deleting older ones
 
 | `xpack.security.audit.appender.kind`
-| Required. Specifies where audit logs should be written to. Allowed values are `console` or `file`.
-|===
+| Required. Specifies where audit logs should be written to. Allowed values are `console`, `file` or `rolling-file`. 
+
+Refer to <<audit-logging-file-appender, file appender>> and <<audit-logging-rolling-file-appender, rolling file appender>> for appender specific settings.
 
+| `xpack.security.audit.appender.layout.kind`
+| Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
+
+Refer to <<audit-logging-pattern-layout, pattern layout>> for layout specific settings.
+
+2+a| 
 [float]
 [[audit-logging-file-appender]]
 ===== File appender
 
-The file appender can be configured using the following settings:
+The `file` appender logs to a file and can be configured using the following settings:
 
-[cols="2*<"]
-|===
 | `xpack.security.audit.appender.path`
 | Required. Full file path the log file should be written to.
 
-| `xpack.security.audit.appender.layout.kind`
-| Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
-|===
+2+a| 
+[float]
+[[audit-logging-rolling-file-appender]]
+===== Rolling file appender
+
+The `rolling-file` appender logs to a file and rotates it using a rolling strategy, when a particular policy is triggered:
+
+| `xpack.security.audit.appender.path`
+| Required. Full file path the log file should be written to.
+
+| `xpack.security.audit.appender.policy.kind`
+| Specifies when a rollover should occur. Allowed values are `size-limit` or `time-interval`. *Default:* `time-interval`.
+
+Refer to <<audit-logging-size-limit-policy, size limit policy>> and <<audit-logging-time-interval-policy, time interval policy>> for policy specific settings.
+| `xpack.security.audit.appender.strategy.kind`
+| Specifies how the rollover should occur. Only allowed value is currently `numeric`. *Default:* `numeric`
+
+Refer to <<audit-logging-numeric-strategy, numeric strategy>> for strategy specific settings.
+
+2+a| 
+[float]
+[[audit-logging-size-limit-policy]]
+===== Size limit triggering policy
+
+The `size-limit` triggering policy will suffix the log file with a given pattern when rolling over, and will limit the number of rolled files to a fixed amount:
```

Comment:
Quick question - this sentence describes what a "strategy" will do, doesn't it? (it's actually quite similar to the description of `Numeric rolling strategy`). Shouldn't it say sth like "The `size-limit` triggering policy will rotate the file once a certain size is reached"

Reply:
Good catch - thanks! Bad copy and paste job ;)